### ISSUE TITLE
Tagify in mode select: to just accept values in the whitelist

### ISSUE THREAD

**Issue by mikeazores** [yairEO/tagify#817]
Hi! I think I'm having an issue very similar to #466 rusing Tagify with mode select. 

You can check here a [demo page](https://codepen.io/mikeazores/pen/OJpywBL?editors=1111).

If I select an item, let's say, "second option", the item is selected just fine. However, after selecting it, If I modify the value to "second op" or any other value, the item remains selected and valid. What I would like to achieve is all the items not in whitelist to be considered invalid and removed from the input box. 

In summary, what I would like to achieve with Tagify is to behave like a normal select input field with the ability to search for the allowed options.  I would like the select box to accept only as valid the values in whitelist (and allow an empty value if not required is marked in Tagify options). Is this achievable?

_Originally posted by @mikeazores in https://github.com/yairEO/tagify/issues/736#issuecomment-838590593_

**Comment by mikeazores** [comment]
I read again the documentation and I guess this behavior is by design. If single value mode is enabled, we can type text as values.

Can anyone advise me if Tagify can be customized easily to make single value mode allow just values from whitelist?

**Comment by yairEO** [comment]
Related to this bug - #818
